### PR TITLE
Fix(Multiple Ped Spawning)

### PR DESCRIPTION
**Pull request by itzhapp**
Right now the ped is networked and will spawn for each client joining, resulting multiple peds.

### CODE REVIEW

**Comment by trclassic92** [comment]
Have you tested this with anyone from your community?

File: client.lua [hunk 1]
```diff
@@ -36,7 +36,7 @@ CreateThread(function()
         while ( not HasModelLoaded( GetHashKey(PedModel) ) ) do
             Wait(1)
         end
-        TRClassicBlackMarketPed = CreatePed(1, PedHash, Coords, true, true)
+        TRClassicBlackMarketPed = CreatePed(1, PedHash, Coords, false, false)
```
Comment (trclassic92) [inline client.lua:39]: `TRClassicBlackMarketPed = CreatePed(1, PedHash, Coords, false, false)`

I suggest that you change it to this

`TRClassicBlackMarketPed = CreatePed(1, PedHash, Coords, false, true)`

If you are able to test this change and let me know if it works then I will push the PR into the main resource 😄

**Review by trclassic92 — CHANGES_REQUESTED** [review]
false, true)

**Comment by itzhapp** [comment]
Tested with community

**Comment by lordfalcon22** [comment]
Tested on my community server and can confirm this fixes the problem with multiple peds

**Pull request merged**